### ISSUE TITLE
Limiting CPU usage

### ISSUE THREAD

**Issue by adamkglaser** [PreibischLab/BigStitcher#48]
Hello,
We are running BigStitcher on the Administrator account of our lab server (running Windows Server 2016). In Fiji, I have limited the resources to use only 50% of the server's CPU and RAM. When I run the fusion step in BigStitcher, I see that the RAM is limited, however the CPU does **not** seem to be limited. And it always uses all cores on the server. Is this a bug - or is there another way to explicitly limit how much CPU BigStitcher uses?
Thanks!
Adam

**Comment by StephanPreibisch** [comment]
Hi, did you set it in edit options memory & threads? How does windows show hyper threading?

**Comment by adamkglaser** [comment]
Yep I set the options under edit -> memory & threads. This might be a naive question, but how do I check if Windows is doing hyper threading? I assume it is, our server has 2x CPUs at 8 cores each, and when I open resource monitor there are 16 CPUs on Nodes 1 and 2 (so 32 CPUs total).
